notebooks/components.py

```diff
@@ -76,24 +79,33 @@ class Table:
     def __init__(self, driver, identifier):
         self.element = get_element_by_identifier(driver, self.xpath, identifier)
 
-    def first_row(self):
+    def row_values(self):
         table_values = []
-        for val in get_children_elements(self.element, 'tbody//tr//td'):
```

Comment:
I suggest refactoring the logic into return a 2d list(row by column) because there's no param for specifying which row to get from, so feels more intuitive to have a 2d structure:
```
        table_values = []
        for row in get_children_elements(self.element, 'tr'):
            row_values = []
            for val in get_children_elements(self.element, 'td'):
                # This query returns a bunch of empty values so we should just keep the values
                if val.text:
                    row_values.append(val.text)
            table_values.append(row_values)
```

so we'd use it like this instead:
```
row = app.table('my-donations-table').row_values()
assert row[0][5] == 'Active'
```